pages/home/passkeys-safe.mdx

```diff
@@ -0,0 +1,123 @@
+
+### `SafeWebAuthnSignerSingleton`
+
+`SafeWebAuthnSignerSingleton` is a singleton contract that implements the ERC-1271 interface to support signature verification. It enables signature data to be forwarded from a Safe to the `WebAuthn` library. This contract expects the caller to append public key coordinates and the verifier address (inspired by [ERC-2771](https://eips.ethereum.org/EIPS/eip-2771)).
+
+### `SafeWebAuthnSignerFactory`
+
+The `SafeWebAuthnSignerFactory` contract deploys the `SafeWebAuthnSignerProxy` contract with the public key coordinates and verifier information. The factory contract also supports signature verification for the public key and signature information without deploying the signer contract, which is used during the validation of ERC-4337 user operations by the experimental `SafeSignerLaunchpad` contract. 
+
+New signers can be deployed using the [ISafeSignerFactory](https://github.com/safe-global/safe-modules/blob/466a9b8ef169003c5df856c6ecd295e6ecb9e99d/modules/passkey/contracts/interfaces/ISafeSignerFactory.sol) interface and this factory contract address.
+
+### `WebAuthn`
+
+This library generates a signing message, hashing it, and forwards the call to the verifier contract. The `WebAuthn` library defines a `Signature` struct containing `authenticatorData` and `clientDataFields`, followed by the ECDSA signature's `r` and `s` components. 
+
+The `authenticatorData` and `clientDataFields` are required for generating the signing message. The `bytes` signature received in the `verifySignature(...)` function is cast to the `Signature` struct, so the caller has to take into account formatting the signature bytes as expected by the `WebAuthn` library.
+The code snippet below shows signature encoding for verification using the WebAuthn library.
+
+```
+bytes authenticatorData = ...;
+string clientDataFields = ...;
+uint256 r = ...;
+uint256 s = ...;
+// Encode the signature data
+bytes memory signature = abi.encode(authenticatorData, clientDataFields, r, s);
+```
+
+### `P256`
+
+`P256` is a library for P256 signature verification with contracts that follow the EIP-7212 EC verify precompile interface. This library defines a custom type `Verifiers`, which encodes two addresses into a single `uint176`. The first address (2 bytes) is a precompile address dedicated to verification, and the second (20 bytes) is a fallback address. 
+
+This setup allows the library to support networks where the precompile is not yet available. It seamlessly transitions to the precompile when it becomes active while relying on a fallback contract address in the meantime.
+
+## Setup and execution flow
```

Comment:
I just realized this diagram is still here and should be removed. It was related to the deprecated Launchpad contract.